recipes/CommonVoice/ASR/transformer/README.md

```diff
@@ -12,12 +18,31 @@ Here is a list of the different languages that we tested within the CommonVoice
 with our transformers:
 - French
 
+For Whisper finetuning, here is list of the different language that we tested  within the CommonVoice dataset:
+- Hindi
+- Arabic
+- Persian
+- Serbian
+- Mongolian
+- French
+
+
 # Results
 
 | Language | Release | hyperparams file | LM | Val. CER | Val. WER | Test CER | Test WER | Model link | GPUs |
 | ------------- |:-------------:|:---------------------------:| -----:| -----:| -----:| -----:| -----:| :-----------:| :-----------:|
 | French | 2020-06-22 | train_fr.yaml | No | 5.15 | 17.80 | 6.01 | 19.21 | [model](https://drive.google.com/drive/folders/12ny6daoz1Ze1MmgLrsqf352AXvhwob6d?usp=sharing) | 1xV100 16GB |
 
+## Whisper Finetuning Result:
+Following tabl econtains whisper-finetuning results for 1 epoch using whisper_large model when freezing encoder and only fine-tune whisper-decoder.
+| Language | Release | hyperparams file | LM | Val. CER | Val. WER | Test CER | Test WER |Zero-shot Test WER  | GPUs |
```

Comment:
Please add the HuggingFace link/model link as well. (you will need to upload your models on the Gdrive see:https://github.com/speechbrain/speechbrain/tree/develop/recipes/CommonVoice/ASR/seq2seq)

Reply:
I will write a whisper interface to add the model on the HF by end of this week.

Reply:
If you need some help do not hesitate to drop a message :-)

Reply:
Thanks. I have done that the only problem is uploading the models takes long time. I am in process of uploading them.

Reply:
done